docs/topics/introduction.md

```diff
@@ -209,19 +216,24 @@ Run `modelCheckingTest()`. The test will fail with the following output:
 
 ```text
 = Invalid execution results =
-Init part:
-[addLast(22): void]
-Parallel part:
-| pollFirst(): 22 | addFirst(8): void       |
-|                 | peekLast():  22   [-,1] |
+| ---------------------------------------- |
+|      Thread 1     |       Thread 2       |
+| ---------------------------------------- |
+| addLast(22): void |                      |
```

Comment:
Are you sure that `addLast(22)` is generated? The arguments should be smaller.

Reply:
This is the same scenario as it was before this PR. I just changed the format of the output to a new one.

Reply:
I opened an issue about keeping the docs up-to-date #200.

Reply:
Thanks!